src/CodeGenerator/CodeGeneratorConfig.php

```diff
@@ -32,4 +33,14 @@ private function getCommandTemplateContent(string $filename): string
     {
         return file_get_contents(__DIR__ . '/Infrastructure/Template/Command/' . $filename);
     }
+
+    public function getComposerJsonContentAsArray(): array
+    {
+        $filename = Config::getApplicationRootDir() . '/composer.json';
+        if (!file_exists($filename)) {
+            return [];
```

Comment:
Good point. I am not sure what might be the best answer for this question in this context... But I guess you're right, at least for now, there is no way this will work without composer.json file. I will change it and let see.